<?xml version="1.0" ?>
<launch>
  <!-- filter -->
  <node pkg="radar_squat_leg_tracker" type="otsu_filter" name="otsu_filter" output="screen">
    <rosparam command="load" file="$(find radar_squat_leg_tracker)/config/otsu_filter_config.yaml"/>
  </node>

  <!-- detector -->
  <node pkg="radar_squat_leg_tracker" type="human_detector_node" name="radar_human_detector" output="screen">
    <rosparam command="load" file="$(find radar_squat_leg_tracker)/config/human_detector_config.yaml" subst_value="true" />
  </node>

  <!-- tracker -->
  <node pkg="radar_squat_leg_tracker" type="human_tracker_node" name="radar_human_tracker" output="screen" >
    <rosparam command="load" file="$(find radar_squat_leg_tracker)/config/human_tracker_config.yaml"/>
  </node>

  <!-- run rviz -->
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find radar_squat_leg_tracker)/config/rviz/radar_human_tracker.rviz"/>

</launch>
